test/invariant/DelegationInvariants.t.sol

```diff
@@ -105,58 +102,4 @@ contract DelegationInvariants is Fixture {
             }
         }
     }
-
```

Comment:
weren't we supposed to put this in another test script without the timestamp helper, but keep the timestamp helper for the other tests